lib/active_support/cache/memcached_store.rb

```diff
@@ -266,8 +244,14 @@ def expiration(options)
         expires_in
       end
 
-      def log_exception(e)
+      def handle_exceptions(return_value_on_error:, on_miss: return_value_on_error)
```

Comment:
Nitpick: it would be nice if the keyword argument names were consistent: `on_error`, `on_miss` or `return_value_on_error`, `return_value_on_miss`.